### DISCUSSION TITLE
I want to delete my domain and unlink account

### DISCUSSION THREAD

**Discussion by Cryptic1526** [is-a-dev/register#9926] [Support] [answered]
I made the domain because my project leader told me to find a free domain service, but we ended up using a paid service and I forgot about this until I decided to search up my email on google and it came up with https://data.is-a.dev/ with my email. I just wanted to know how I can delete the domain I have and unlink my information. I'm pretty new to github so please bear with me!

**Comment by wdhdev** [comment, marked as answer]
You just need to find and delete your subdomain file in your forked repository and open a PR.

**Reply by Cryptic1526** [reply]
Eh I think I did it... Could you see if I did it correctly? https://github.com/is-a-dev/register/pull/9972
